Connection/WSConnector.cs

```diff
@@ -255,6 +255,7 @@ public static string FixupURL(string URL)
             //  platforms.
             else if (URL.StartsWith("https://stream-tls10."))
             {
+                Log.Warning("WSConnector", "Streaming with TLS 1.0 will be  deprecated in next Major release!");
```

Comment:
how about this (if accurate):
> Deprecated: Support for TLS 1.0 is deprecated and will not be supported in the next major release of the SDK.

Reply:
Sounds good, one question, should we something other than `Support for TLS 1.0 is deprecated` since it's not deprecated yet.

Reply:
Sounds good, one question, should we something other than `Support for TLS 1.0 is deprecated` since it's not deprecated yet.

Reply:
Sounds good, one question, should we something other than `Support for TLS 1.0 is deprecated` since it's not deprecated yet.

Reply:
deprecated means that we don't want them to use it, not that it's removed. So if you're going to remove in the next major release, then we'd call it deprecated now. Looking at the README, it seems that only Dallas supports TLS 1. 

But if you're going to deprecate in the next release (and then remove even later), try this on for size.
> Support for TLS 1.0 is deprecated and will be removed in the next major release of the SDK. Update to Unity version 2018.2 or later to support TLS 1.2.

If it is really deprecated now and removed in next version, this might be better too
> Deprecated: Support for TLS 1.0 will be deprecated in the next major release of the SDK and will support TLS 1.2. Unity version 2018.2 and later supports TLS 1.2.

Reply:
Personally I find that last message confusing. The "Deprecated" at the beginning is nice an clear -- this thing is deprecated.  But then it goes on to say "Support for TLS 1.0 _will be deprecated_ in the next major release".  So is it deprecated now or not until the next major release?

My suggestion is:
> Deprecated: Support for TLS 1.0 will be _removed_ in the next major release of the SDK ...

Reply:
I'm happy for the vote for the  `deprecated:` lead-in. Mike's version is short too.